charts/posthog/values.yaml

```diff
@@ -242,6 +241,34 @@ email:
   # existingSecret: secret-name
   # existingSecretKey: smtp-password
 
+# SAML
+saml:
+  # -- Whether password-based login is disabled and users automatically redirected to SAML login. Requires SAML to be properly configured.
+  enforced: false
+  # -- Whether SAML should be completely disabled. If set at build time, this will also prevent SAML dependencies from being installed.
+  disabled: false
+  # -- Entity ID from your SAML IdP.
+  # entity_id: id-from-idp-5f9d4e-47ca-5080
+  entity_id:
+  # -- Assertion Consumer Service URL from your SAML IdP.
+  # acs_url: https://mysamlidp.com/saml2
```

Comment:
Not sure if we need to escape this somehow, would appreciate some input

Reply:
wdym? What needs escaping?

Reply:
Not sure if the `/` or `:` would need escaping for some reason ?

Reply:
generally all strings in quotes is the suggestion from helm https://helm.sh/docs/chart_best_practices/values/#make-types-clear

Reply:
Escaped